config/src/config/consensus_config.rs

```diff
@@ -8,8 +8,8 @@ use std::path::PathBuf;
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
 pub struct ConsensusConfig {
-    pub max_block_size: u64,
     pub contiguous_rounds: u32,
+    pub max_block_size: u64,
```

Comment:
IIUC by swapping these, but not swapping these in config files that we already have, we now have configs that will load as:

* contiguous_rounds = previous_max_block_size
* max_block_size = previous_contiguous_round

and this is in part facilitated by the fact that u32 and u64 are (de)serialized the same way.

That's probably the only danger there is in refactoring the config module.

Reply:
that's not the case, all the configs we have are named (yaml) versus lcs. If we did use lcs or some other binary / anonymous field-based config, you are 100% right.

Reply:
oh my brain did fart indeed